File: LLM-Translations/GPT-4/codenet/C/Go/s606068991.go

package main

import (
	"fmt"
)

func white(n int, A []int) {
	for i := 0; i < n-1; i++ {
		fmt.Printf("%d ", A[i])
	}
	fmt.Printf("%d\n", A[n-1])
}

func insertionSort(n int, A []int) {
	var i, j, a int
	for i = 0; i < n; i++ {
		for j = 0; j < i; j++ {
			if A[i] < A[j] {
				a = A[i]
				A[i] = A[j]
				A[j] = a
			}
		}
		white(n, A)
	}
}

func main() {
	var i, n int
	var A = make([]int, 100)
	fmt.Scanf("%d", &n)
	for i = 0; i < n; i++ {
		fmt.Scanf("%d", &A[i])
	}
	insertionSort(n, A[:n])
}

// 

File: LLM-Translations/GPT-4/codenet/C/Java/s606068991.java

import java.util.Scanner;

public class s606068991 {

    public static void white(int n, int[] A) {
        for (int i = 0; i < n - 1; i++) {
            System.out.print(A[i] + " ");
        }
        System.out.println(A[n - 1]);
    }

    public static void insertionSort(int n, int[] A) {
        int i, j, a;
        for (i = 0; i < n; i++) {
            for (j = 0; j < i; j++) {
                if (A[i] < A[j]) {
                    a = A[i];
                    A[i] = A[j];
                    A[j] = a;
                }
            }
            white(n, A);
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int i, n;
        n = scanner.nextInt();
        int[] A = new int[100];
        for (i = 0; i < n; i++) {
            A[i] = scanner.nextInt();
        }
        insertionSort(n, A);
    }
}

// 
